src/client/src/orangehrmBuzzPlugin/components/BasePost.vue

```diff
@@ -1,33 +1,76 @@
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+
 <template>
-  <!-- TODO: Placeholder component -->
-  <oxd-sheet class="orangehrm-buzz-post">
-    <div class="orangehrm-buzz-post-profile">
-      <img
-        alt="profile picture"
-        class="employee-image"
-        :src="`../pim/viewPhoto/empNumber/${employee.empNumber}`"
-      />
-      <oxd-text type="card-title">
-        {{ employeeFullName }}
-      </oxd-text>
+  <oxd-sheet :gutters="false" type="white" class="orangehrm-buzz">
+    <div class="orangehrm-buzz-post">
+      <div class="orangehrm-buzz-post-header">
+        <div class="orangehrm-buzz-post-header-details">
+          <div class="orangehrm-buzz-post-profile-image">
+            <img
+              alt="profile picture"
+              class="employee-image"
+              :src="`../pim/viewPhoto/empNumber/${employeeId}`"
+            />
+          </div>
+          <div class="orangehrm-buzz-post-header-text">
+            <oxd-text tag="p" class="orangehrm-buzz-post-emp-name">
+              {{ employeeFullName }}
+            </oxd-text>
+            <oxd-text tag="p" class="orangehrm-buzz-post-time">
+              {{ postedTime }}
+            </oxd-text>
+          </div>
+        </div>
+        <div class="orangehrm-buzz-post-header-config">
+          <oxd-dropdown icon="three-dots" :options="dropdownOptions" />
+        </div>
+      </div>
+      <oxd-divider />
+    </div>
+    <div class="orangehrm-buzz-post-body">
+      <oxd-text>{{ content }}</oxd-text>
+      <slot name="body"></slot>
     </div>
-    <oxd-divider></oxd-divider>
-    <oxd-text type="card-title">{{ content }}</oxd-text>
+    <div class="orangehrm-buzz-post-footer">
+      <div class="orangehrm-buzz-post-footer-left">
+        <slot name="actionButton"> </slot>
+      </div>
+      <slot name="postStatus"> </slot>
+    </div>
+    <slot name="comments"></slot>
   </oxd-sheet>
 </template>
-
 <script>
 import {computed} from 'vue';
 import Sheet from '@ohrm/oxd/core/components/Sheet/Sheet';
+import {APIService} from '@/core/util/services/api.service';
 import useEmployeeNameTranslate from '@/core/util/composable/useEmployeeNameTranslate';
+import Dropdown from '@ohrm/oxd/core/components/CardTable/Cell/Dropdown.vue';
```

Comment:
Can't use this component for this purpose. It's a internal component of cardtable. If you need something like this, implement a local component.

Reply:
https://github.com/orangehrm/orangehrm/pull/1497 FIxes add here